src/schema/bookmarks.ts

```diff
@@ -382,6 +385,12 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       { id, listId }: { id: string; listId?: string },
       ctx: AuthContext,
     ): Promise<GQLEmptyResponse> => {
+      if (!ctx.isPlus) {
+        throw new ForbiddenError(
+          'You need to be a Plus member to use this feature',
```

Comment:
let's export this to errors with other error labels

Reply:
Update ✔️ 🙏